README.md

```diff
@@ -34,4 +34,25 @@ that's built into gxi, so please don't use `cargo` to install gxi, as that won't
 from a different rev can lead to very weird bugs.
 
 
-After these steps you should be able to run gxi simply by invoking `gxi`
\ No newline at end of file
+After these steps you should be able to run gxi simply by invoking `gxi`
+
+### Installation on Arch/Manjaro
+
+The package for Arch Linux is [gxi-git](https://aur.archlinux.org/packages/gxi-git/). Building and installing
+(including dependencies) can be accomplished with:
+
+```sh
+yaourt -Sy gxi-git
+```
+
+Alternatively use ``makepkg``:
```

Comment:
Although not a blocker, why have two '`' here?

Reply:
Just a typo, fixed.

Reply:
Thanks :)